README.md

```diff
@@ -114,7 +114,9 @@ Configures Providers. For example, `$locationProvider.html5Mode(true);`.
 
 ### Constants
 
-[Constant variables](http://en.wikipedia.org/wiki/Constant_(computer_programming)). For example, `export var API = 'https://api.gocardless.com';`. Although JavaScript does not yet support constants, naming a variable in all uppercase is a convention that denotes the variable should not be mutated.
+[Constants](http://en.wikipedia.org/wiki/Constant_(computer_programming)). For example, `export const API = 'https://api.gocardless.com';`.
+
+Although JavaScript does not yet support constants, we run our application through Traceur, which supports `const`. Naming the variable in uppercase is a convention that helps recognise if a variable is a constant.
```

Comment:
maybe worth noting when to use uppercase and when not to, given we now use `const` whenever we can